services/static-webserver/client/source/class/osparc/data/model/User.js

```diff
@@ -32,8 +32,18 @@ qx.Class.define("osparc.data.model.User", {
     const groupId = ("gid" in userData) ? parseInt(userData["gid"]) : parseInt(userData["groupId"]);
     const username = userData["userName"];
     const email = ("login" in userData) ? userData["login"] : userData["email"];
-    const firstName = ("first_name" in userData) ? userData["first_name"] : userData["firstName"];
-    const lastName = ("last_name" in userData) ? userData["last_name"] : userData["lastName"];
+    let firstName = "";
```

Comment:
When **filling a form**, if there is a field missing/wrong I a use to a different UX. 
- pop up: warns about one or more fields wrong/missing
- highlight in red fields that are wrong in the form with the error message

![image](https://github.com/user-attachments/assets/9599126b-e9bf-4078-bb25-91f46b8861cb)


PS: this could be also reused to map validation errors from the backend to the front-end

Reply:
Implemented:
![Name](https://github.com/user-attachments/assets/3217d9c3-3741-4b45-b34e-cb058b536a0a)